packages/e2e/detox/integration/common/shared.ts

```diff
@@ -1,6 +1,19 @@
 import { Given, Then, When } from '@cucumber/cucumber';
 import { by, element, expect } from 'detox';
 
+export const typeInInputField = async (
+  inputFieldMatcher: Detox.NativeMatcher,
+  text: string
+) => {
+  const inputField = element(inputFieldMatcher);
+  await device.disableSynchronization();
+  // replaceText is much faster than typeText
+  // downside: doesn't trigger text input callbacks (such as validations) so we need to tap return key
```

Comment:
To confirm, we will not be able to validate on `blur` events?

Reply:
No, this explains why we need to add the command to tap the return key 2 lines below. Taping return triggers the callbacks. Open to rephrasing :)

Reply:
Still a bit confused, will `change` events still trigger?

Reply:
Yes, they will trigger after text in the field is replaced. So not character by character. We have e2es that test validation errors and are passing. did you have specific concerns?

Reply:
Yes, want to ensure that this update will not be problematic for validation events in the future. IIRC this is only need for Android?

Reply:
Synced offline, will follow up with single validation test that exercises typing to be covered.